libcobblersignatures/cli.py

```diff
@@ -5,6 +5,7 @@
 """
 
 import questionary
+from questionary import Validator, ValidationError
```

Comment:
If `questionary` is already imported why re-importing and not simply using `questionary.Validator` and so on later?

Reply:
That is indeed true. Should be fixed by c8cf3a5